cli/cmd/compliance_gcp.go

```diff
@@ -32,10 +32,46 @@ import (
 )
 
 var (
+	// complianceGcpListCmd represents the list sub-command inside the gcp command
+	complianceGcpListCmd = &cobra.Command{
+		Use:   "list",
+		Short: "list gcp projects and organizations",
+		Long:  `List all GCP projects and organization IDs.`,
+		RunE: func(_ *cobra.Command, args []string) error {
+			var (
+				gcpAccounts   []gcpAccount
+				response, err = cli.LwApi.Integrations.ListGcpCfg()
+				rows          [][]string
+			)
+
+			if err != nil {
+				return errors.Wrap(err, "unable to list gcp projects/organizations")
+			}
+
+			for _, gcp := range response.Data {
+				gcpIntegration := gcpAccount{}
+				// if organization account, fetch the project ids
+				if gcp.Data.IDType == "ORGANIZATION" {
+					gcpAccounts = append(gcpAccounts, getGcpAccounts(gcpIntegration.OrganizationID)...)
```

Comment:
`gcpIntegration.OrganizationID` will always be an empty string since you just initialized at
line `52`, I think you meant to use `gcp.Data.ID` instead.
```suggestion
					gcpAccounts = append(gcpAccounts, getGcpAccounts(gcp.Data.ID)...)
```

Still, this shows that we need to test this a bit more, could you please fix this issue and make sure we
can add more tests?

Reply:
Did some more tests locally. At the minute while we're making changes to our integration test automation I don't want to add a dependency on having a gcp org account(yet). But I'll make a ticket to add tf and integration test for this.